### PR TITLE
add centos-streaming9-mlnet-helix docker image

### PR DESCRIPTION
**Pull request by LittleLittleCloud**
Fixes https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/1131

### CODE REVIEW

File: src/centos/stream9/mlnet/helix/Dockerfile
Comment (mthalman) [inline src/centos/stream9/mlnet/helix/Dockerfile:1]: Please apply the same pattern of changes that exist in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1122 to switch to `venv`.

File: src/centos/manifest.json [hunk 1]
```diff
@@ -28,6 +28,22 @@
...
+            {
+              "dockerfile": "src/centos/stream9/mlnet/helix",
+              "os": "linux",
+              "osVersion": "centos-stream9",
+              "tags": {
+                "centos-stream9-mlnet-helix-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "centos-stream9-mlnet-helix$(FloatingTagSuffix)": {},
+                "centos-stream9-mlnet-helix-local": {
+                  "isLocal": true
+                }
```
[the remark this answers is not in the document]
Comment (LittleLittleCloud) [reply src/centos/manifest.json:43]: Do I need this flag to generate the image tag without gitcommit/timestamp in its name

[the remark this answers is not in the document]
Comment (michaelgsharp) [reply src/centos/manifest.json]: yeah, but do you need the image tag without that stuff for this image?

[the remark this answers is not in the document]
Comment (mthalman) [reply src/centos/manifest.json]: > Do I need this flag to generate the image tag without gitcommit/timestamp in its name

That's not what this flag means. `isLocal` means it only creates that tag locally on the build machine. It doesn't push it to the container registry. This is the tag which excludes the commit/timestamp: `centos-stream9-mlnet-helix$(FloatingTagSuffix)`.

**Review by mthalman — APPROVED** [review]

File: src/centos/stream9/mlnet/helix/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        swig \
+        tar \
+        wget \
+        which \
+        xz \
+        zlib-devel \
+    && \
+    dnf clean all
+
+RUN dnf install perl-FindBin --setopt tsflags=nodocs --refresh -y
```
[the remark this answers is not in the document]
Comment (LittleLittleCloud) [reply src/centos/stream9/mlnet/helix/Dockerfile:61]: I got an error when running L68 reminding that `perl-FindBin` is missing, that's why I add L61

**Pull request merged**